src/core/client.ts

```diff
@@ -131,14 +144,20 @@ export default class Client {
 
     if (objectIsEmpty(notice)) { return false }
 
+    const noticeTags = this.__constructTags(notice.tags)
+    const contextTags = this.__constructTags(this.__context["tags"])
+    const configTags = this.__constructTags(this.config.tags)
+    const tags = noticeTags.concat(contextTags).concat(configTags)
```

Comment:
I forget, does concat remove duplicates?

Reply:
Survey says no!

```js
["1"].concat(["1"])
// => ["1", "1"]
```

Reply:
@joshuap would you like it to remove duplicates? I could easily run a `.filter()`

Reply:
@joshuap I totally forgot about the `Set` data structure and used that rather than a filter to extract unique tags.

Reply:
So i tried using `const uniqueTags = Array.from(new Set(tags))` and it broke CI. Interesting...

Reply:
It might be that `Set` is not available in IE 11?

Reply:
> @joshuap would you like it to remove duplicates? I could easily run a `.filter()`

Yeah, I would like to remove duplicates. 👍

Reply:
Bingo. Set is supported in IE11, but cannot be converted back to an array with an iterable.